v20.2/disaster-recovery.md

```diff
@@ -0,0 +1,333 @@
+      <th>Consequence</th>
+      <th>Action to Take</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td style="color:#46a417"><b>1 Disk</td>
+      <td style="color:#228B22"><b>√</b></td>
+      <td rowspan="5">Under-replicated data. Less resources for workload.</td>
+      <td><a href="start-node.html">Restart the node</a> with a new disk.</td>
+    </tr>
+      <td style="color:#46a417"><b>1 Server</td>
+      <td style="color:#228B22"><b>√</b></td>
+      <td rowspan="3">If using a Kubenetes StatefulSet, a node will be restarted automatically. <br><br>If the server, rack, or AZ becomes available check the <a href="admin-ui-overview-dashboard.html">Overview dashboard</a> on the Admin UI:
+      <br><br>- If the down server is marked <b>Suspect</b>, try <a href="start-node.html">restarting the node</a>.
+      <br>- If the down server is marked <b>Dead</b>, <a href="remove-nodes.html">decommission the node</a> and add a new server. If you try to rejoin the same decommissioned node back into the server, you should wipe the store path before rejoining.</td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>1 Rack</b></td>
+      <td style="color:#228B22"><b>√</b></td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>1 AZ</b></td>
+      <td style="color:#228B22"><b>√</b></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>1 Region</b></td>
+      <td style="color:#228B22"><b>√</b></td>
+      <td>Check the <a href="admin-ui-overview-dashboard.html">Overview dashboard</a> on the Admin UI. If servers are marked <b>Dead</b>, <a href="remove-nodes.html">decommission the nodes</a> and add 3 new servers in a new region.</td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>2 or More Regions</b></td>
+      <td style="color:#FF0000"><b>X</b></td>
+      <td>Cluster will be unavailable. <br><br>Potential data loss between last backup and time of outage if region and servers did not come back online. This would be a considerable disaster (i.e., 2 or more data centers destroyed).</td>
+      <td>When the regions comes back online, try <a href="cockroach-start.html">restarting the nodes</a> in the cluster. <br><br>If the regions do not come back online and servers are lost or destroyed, try <a href="restore.html">restoring the latest cluster backup</a> into a new cluster.</td>
+    </tr>
+  </tbody>
+</table>
+
+<span style="color:#228B22"><b>√</b></span> = Available
+<span style="color:#FF0000"><b>X</b></span> = Outage
+
+## Data failure
+
+When dealing with data failure due to bad actors, rogue applications, or data corruption, domain expertise is required for long-term solutions. However, there are a few options for short term remediation that you can take:
+
+#### Basic recovery actions
+
+- Using your backup file, [restore to a point in time](backup-and-restore-advanced-options.html#point-in-time-restore) for the corrupted data to a point where you are certain there was no corruption.
```

Comment:
We might want Core vs. Enterprise tags on these. In 20.2 Core will have backup but not Point in time restore (PITR) or revision history backups. They CAN take a backup immediately, backdate it to the point they want to restore to, take that backup and restore it.